app/mailers/rrd_completed_mailer.rb

```diff
@@ -3,8 +3,8 @@
 class RrdCompletedMailer < ApplicationMailer
   def build(submission)
     @submission = submission
-    @disability = submission.form.dig('form526', 'form526', 'disabilities')&.first
-    @disability_code = @disability['diagnosticCode'] if @disability
+    @disability_struct = RapidReadyForDecision::Constants.first_disability(submission) || {}
+    @disability_code = @disability_struct['code']
```

Comment:
Use `disability_struct` so we can use the `disability_struct[:label]` in the erb file.